zerver/openapi/zulip.yaml

```diff
@@ -18647,9 +18647,35 @@ paths:
 
                                 **Changes**: New in Zulip 5.0 (feature level 93).
                             can_mention_group:
-                              type: integer
+                              oneOf:
+                                - type: integer
+                                - type: object
+                                  additionalProperties: false
+                                  properties:
+                                    direct_members:
+                                      description: |
+                                        The list of user IDs that are direct members of an
+                                        anonymous user group.
+                                      type: array
+                                      items:
+                                        type: integer
+                                    direct_subgroups:
+                                      description: |
+                                        The list of user group IDs that are direct subgroups
+                                        of an anonymous user group.
+                                      type: array
+                                      items:
+                                        type: integer
                               description: |
-                                ID of the user group whose members are allowed to mention the group.
+                                User group whose members are allowed to mention the group.
+
+                                This can either be ID of a named user group or an object
+                                containing direct members and direct subgroups of an
+                                anonymous user group.
```

Comment:
I still think this can be improved. Basically I am not sure how to represent the unnamed user groups - should we mention  "anonymous" or "unnamed". Or we just altogether skip mentioning anything about that and just mention something like a "set of users and groups who can mention this group", but then the setting name contains "_group" which represents that a group is allowed to mention any group.

Reply:
I think something like this should work:

> Either the ID of a named user group that has permission to do this operation, or an object describing the set of users and groups who have permission to do this operation.

Reply:
Updated the text and also replaced "to do this operation" with "to mention the group".